packages/office-ui-fabric-react/src/packages/react-commanding/components/Nav/Nav.doc.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```

Comment:
I am guessing using VS code extensions like Git Lens that is displaying inline who last edited the line will be useless now 😞